### ISSUE TITLE
Imported sources

### ISSUE THREAD

**Issue by Jessica-Chen85** [otwin-core/otwin-hybrid#3]
Hi,

 I would like to ask a question about the following imported sources.
You import several sources and I know pandas and numpy are contained in Pycharm.
How about other sources such as 'sklearn.model_selection',  'keras', 'tensorflow', 'plotly'. It seems that those sources are not on the website. May I ask where I can find those sources?

Thank you so much

**Comment by Javihaus** [comment]
Hi Jessica,

you can import as in the notebook:


```
import pandas as pd
import numpy as np
from sklearn.model_selection import train_test_split
import keras
import tensorflow as tf
from keras.models import Sequential
from keras.layers import LSTM, Dense
from keras.optimizers import SGD
from tensorflow.keras.layers import Dropout
from matplotlib import pyplot as plt
import plotly.express as px
from plotly.subplots import make_subplots
import plotly.graph_objects as go
import plotly.figure_factory as ff
```
'''

**Comment by Jessica-Chen85** [comment]
HI, I already copied the following codes in my Pycharm. 

import pandas as pd
import numpy as np
from sklearn.model_selection import train_test_split
import keras
import tensorflow as tf
from keras.models import **Sequential**
from keras.layers import **LSTM, Dense**
from keras.optimizers import **SGD**
from **tensorflow.keras.layers** import **Dropout**
from matplotlib import pyplot as plt
import plotly.express as px
from plotly.subplots import make_subplots
import plotly.graph_objects as go
import plotly.figure_factory as ff

The problem now is that 'Sequential', 'LSTM', 'Dense', 'SGD', 'tensorflow.keras.layers', and 'Dropout'  still show red lines below.
Therefore, I would like to ask where are those models ? 

Thank you very much

**Comment by Jessica-Chen85** [comment]
Hi,

I think that it is the version problem.
I updated my source version and it work.

Thanks
